frontend/src/app/components/ExperimentRowCard/index.js

```diff
@@ -173,11 +173,8 @@ export default class ExperimentRowCard extends React.Component {
     const created = (new Date(experiment.created)).getTime();
     if ((now - created) > MAX_JUST_LAUNCHED_PERIOD) { return false; }
 
-    // If never seen, *do* consider it "just" launched.
-    if (!getExperimentLastSeen(experiment)) { return true; }
-
     // All else fails, don't consider it launched.
```

Comment:
This is just a nit, this comment is no longer correct, but I am going to go ahead and merge this anyway. Open another PR to fix this comment if desired.